x-pack/test/cloud_security_posture_functional/agentless/create_agent.ts

```diff
@@ -68,7 +70,14 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
       expect(await cisIntegration.getFirstCspmIntegrationPageAgentlessIntegration()).to.be(
         integrationPolicyName
       );
-      expect(await cisIntegration.getFirstCspmIntegrationPageAgentlessStatus()).to.be('Pending');
+
+      // wait for eventually Pending or Healthy status
+      // purpose of this retry is to wait for the agent to be created and the status to be updated
+      // not to wait for the agent to be healthy
+      await retry.tryForTime(agentCreationTimeout, async () => {
+        const resStatus = await cisIntegration.getFirstCspmIntegrationPageAgentlessStatus();
+        expect(resStatus == 'Healthy' || resStatus == 'Pending').to.be(true);
+      });
```

Comment:
I honestly don't think an assertion that can have two possible states is going to be reliable in the long-term, having resolve as successful with either 'Pending' or 'Healthy' will lead to a false sensation of coverage when some bad configuration makes it stuck on Pending and we end up not catching it.

Reply:
Initially implemented with a `Healthy` assertion, and I agree with what you've said.
But @seanrathier raised that we do not care about the actual agent state, and we just want to make sure that we have a value that makes sense, which indicates that the agent was initiated (which means I assume that the request trigger worked).
I think that it would be great if you both could sync and decide on something, and I will make the relevant changes (I've been implementing back and forth).

One thing that I would like us to do is to verify that the agent got deployed successfully and it works, I won't go into if this test should verify that or some other test (will let you both decide).